### PR TITLE
Feature send event to bot

### PR DESCRIPTION
**Pull request by vivekkhandelwal55**

### CODE REVIEW

File: example/android/build.gradle [hunk 1]
```diff
@@ -22,6 +22,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```
Comment (sankalp211) [inline example/android/build.gradle:25]: Why this change is required @vivekkhandelwal55 ?

**Review by sankalp211 — CHANGES_REQUESTED** [review]

**Review by rohit-131 — APPROVED** [review]

**Review by sankalp211 — APPROVED** [review]
